Xdl.Internship.Core/Xdl.Internship.Core.DataAccess.MongoDB/ConnectionFactories/ConnectionFactory.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Threading;
+using Microsoft.Extensions.Options;
+using MongoDB.Driver;
+using Xdl.Internship.Core.DataAccess.MongoDB.Settings;
+
+namespace Xdl.Internship.Core.DataAccess.MongoDB.ConnectionFactories
+{
+    public class DefaultConnectionFactory : IConnectionFactory
```

Comment:
pls rename file to DefaultConnectionFactory.cs

Reply:
resolved